lib/licensee/license.rb

```diff
@@ -38,13 +50,23 @@ def find(key, options = {})
       alias [] find
       alias find_by_key find
 
-      def license_dir
```

Comment:
For backwards compatibility, we'll want a class method 

```ruby
def license_dir
  license_dirs.first
end
```

That way, it remains accurate even if overwritten.